cob_hand_bridge/launch/sdhx.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
we don't need this launch file, it's in cob_bringup (see https://github.com/ipa320/cob_robots/pull/753). please remove